evap/staff/templates/staff_semester_view.html

```diff
@@ -64,76 +60,69 @@
     {% endif %}
 
     {% if request.user.is_staff %}
-    <div class="panel panel-default">
-        <div class="panel-heading">
-            <h3 class="panel-title">{% trans "Actions" %}</h3>
+    <div class="card mb-3">
+        <div class="card-header">
+            {% trans "Actions" %}
         </div>
-        <div class="panel-body">
-            <div class="button-menu">
-                <a href="{% url "staff:semester_questionnaire_assign" semester.id %}" class="btn btn-sm btn-default {{ disable_if_archived }}">{% trans "Assign Questionnaires" %}</a>
-                &nbsp;
-                <a href="{% url "staff:semester_todo" semester.id %}" class="btn btn-sm btn-default {{ disable_if_archived }}">{% trans "Todo" %}</a>
-                &nbsp;
-                <a href="{% url "staff:semester_lottery" semester.id %}" class="btn btn-sm btn-default">{% trans "Lottery" %}</a>
-                <div class="btn-group">
-                    <a href="" class="btn btn-sm btn-default disabled">{% trans "Reward points active" %}</a>
-                    <div class="btn-group btn-toggle">
-                        <a href="{% url "rewards:semester_activation" semester.id "on" %}" class="btn btn-sm btn-default {% if rewards_active %}active{% endif %}"><span class="glyphicon glyphicon-ok" aria-hidden="true"></span></a>
-                        <a href="{% url "rewards:semester_activation" semester.id "off" %}" class="btn btn-sm btn-default {% if rewards_active == False %}active{% endif %}"><span class="glyphicon glyphicon-remove" aria-hidden="true"></span></a>
-                    </div>
-                </div>
-                &nbsp;
-                <div class="btn-group">
-                    <a href="" class="btn btn-sm btn-default disabled">{% trans "Grades downloadable" %}</a>
-                    <div class="btn-group btn-toggle">
-                        <a href="{% url "grades:semester_grade_activation" semester.id "on" %}" class="btn btn-sm btn-default {% if grades_downloadable %}active{% endif %}"><span class="glyphicon glyphicon-ok" aria-hidden="true"></span></a>
-                        <a href="{% url "grades:semester_grade_activation" semester.id "off" %}" class="btn btn-sm btn-default {% if grades_downloadable == False %}active{% endif %}"><span class="glyphicon glyphicon-remove" aria-hidden="true"></span></a>
-                    </div>
-                </div>
+        <div class="card-body">
+            <a href="{% url "staff:semester_questionnaire_assign" semester.id %}" class="btn btn-sm btn-light {{ disable_if_archived }}">{% trans "Assign Questionnaires" %}</a>
+            &nbsp;
+            <a href="{% url "staff:semester_todo" semester.id %}" class="btn btn-sm btn-light {{ disable_if_archived }}">{% trans "Todo" %}</a>
+            &nbsp;
+            <a href="{% url "staff:semester_lottery" semester.id %}" class="btn btn-sm btn-light">{% trans "Lottery" %}</a>
```

Comment:
actually... why don't we just remove the lottery?

Reply:
done